geniza/corpus/management/commands/sync_transcriptions.py

```diff
@@ -63,7 +69,12 @@ def handle(self, *args, **options):
             # some files are stubs with no content
             # check if there is no text content; report and skip
             if tei.no_content():
-                self.stdout.write("%s has no text content, skipping" % xmlfile)
+                if self.verbosity >= self.v_normal:
```

Comment:
I'm not familiar with this pattern. Does it provide an advantage over just using `-v` for verbose? (Or I guess for this case it would be more like `-q` for quiet, since the default `v_normal` would write all of these outputs.)

Reply:
django manage command verbosity default parameter is a numeric argument so output can have varying levels of verbosity, and you can say `-v 0`, `-v 1`, `-v 2`, etc — this `v_normal` stuff is a convention I adopted ages ago as a more semantic way to control the output — although I don't think we're using the granularity much in this script, it could let us optionally make the output much more verbose if we wanted

Reply:
I see, makes sense! Didn't know manage commands had a built in verbosity parameter.